config/environments/production.rb

```diff
@@ -81,5 +77,4 @@
   # Do not dump schema after migrations.
   config.active_record.dump_schema_after_migration = false
 
-  config.action_mailer.default_url_options = { host: 'please-change-me.com' }
```

Comment:
Ok cool! You should also make the relevant changes to `staging.rb` then. And you won't need the `letter_opener` and `recipient_interceptor` gems.